frontend/build/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Why do we have this? Seems like a new file?

Reply:
Build folder shouldn't be pushed at all, not sure how it snuck in there.

Removed and updated.

Reply:
thanks!